pkgs/development/python-modules/django-health-check/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pytest-runner
+, sphinx
+, setuptools_scm
+, django
+, redis
+, celery
+, pytest-django
+, pytest-cov
+, pytest
+, mock }:
+
+buildPythonPackage rec {
+  pname = "django-health-check";
+  version = "3.16.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Ht/UkpPM67zin52mCcQH8weu4kCreZq0IBAxNBrnjA8=";
+  };
+
+  propagatedBuildInputs = [
+    pytest-runner
+    sphinx
+    setuptools_scm
```

Comment:
This belongs to nativeBuildInputs and is an alias.

Reply:
Oh I see you meant nativeBuildInputs not checkInputs

Reply:
I think you missed this.